sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/api/Azure.ResourceManager.AppConfiguration.netstandard2.0.cs

```diff
@@ -283,6 +292,49 @@ public partial class DeletedAppConfigurationStoreResource : Azure.ResourceManage
+        public virtual Azure.Response<bool> Exists(string replicaName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<bool>> ExistsAsync(string replicaName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.AppConfiguration.ReplicaResource> Get(string replicaName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<Azure.ResourceManager.AppConfiguration.ReplicaResource> GetAll(string skipToken = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.ResourceManager.AppConfiguration.ReplicaResource> GetAllAsync(string skipToken = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.AppConfiguration.ReplicaResource>> GetAsync(string replicaName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.NullableResponse<Azure.ResourceManager.AppConfiguration.ReplicaResource> GetIfExists(string replicaName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.NullableResponse<Azure.ResourceManager.AppConfiguration.ReplicaResource>> GetIfExistsAsync(string replicaName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        System.Collections.Generic.IAsyncEnumerator<Azure.ResourceManager.AppConfiguration.ReplicaResource> System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.AppConfiguration.ReplicaResource>.GetAsyncEnumerator(System.Threading.CancellationToken cancellationToken) { throw null; }
+        System.Collections.Generic.IEnumerator<Azure.ResourceManager.AppConfiguration.ReplicaResource> System.Collections.Generic.IEnumerable<Azure.ResourceManager.AppConfiguration.ReplicaResource>.GetEnumerator() { throw null; }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+    }
+    public partial class ReplicaData : Azure.ResourceManager.Models.ResourceData, System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.AppConfiguration.ReplicaData>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.AppConfiguration.ReplicaData>
```

Comment:
The name of this set of resource is too generic.
Could it be more specific? I suggest we change it to `AppConfigurationReplica[Data/Resource/Collection]`
To do this, please add an entry in the `rename-mapping` in autorest.md:
```
rename-mapping:
  # existing entries
  Replica: AppConfigurationReplica # this will change three types XXXData, XXXResource and XXXCollection
```

Reply:
Thanks! Added `Replica: AppConfigurationReplica` under the `rename-mapping` and regenerated code.